src/components/PlayerSplit.tsx

```diff
@@ -175,8 +175,8 @@ export class PlayerSplitComponent extends React.Component<PlayerSplitComponentPr
         document.exitFullscreen();
       } else if((document as any).mozCancelFullScreen) {
         (document as any).mozCancelFullScreen();
-      } else if(document.webkitExitFullscreen) {
-        document.webkitExitFullscreen();
+      } else if((document as any).webkitExitFullscreen) {
```

Comment:
If you want to overachieve, you can follow the text at the end of the Typescript release notes to add back webkitExitFullscreen instead of casting to any: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html

Reply:
I added an interface for documents within the PlayerSplit file. I used a similar solution to the one you linked. Let me know if that works

bcbc8fd

Reply:
Looks good.